core/src/main/resources/META-INF/fabric8/kind-fileindicator-mapping-default.adoc

```diff
@@ -0,0 +1,102 @@
+|===
+|Kind |Filename Type
+
+|ConfigMap
+a|`cm`, `configmap`
+
+|CronJob
+a|`cronjob`, `cj`
+
+|ClusterRole
+a|`cr`, `crole`, `clusterrole`
+
+|CustomResourceDefinition
+a|`crd`
+
+|ClusterRoleBinding
+a|`crb`, `clusterrb`
+
+|Deployment
+a|`deployment`
+
+|ImageStream
+a|`is`
+
+|ImageStreamTag
+a|`istag`
+
+|Job
+a|`job`
+
+|LimitRange
+a|`lr`, `limitrange`
+
+|Namespace
+a|`ns`, `clusterrb`
+
+|OAuthClient
+a|`oauthclient`
+
+|PolicyBinding
+a|`pb`
+
+|PersistentVolume
+a|`pv`
+
+|PersistentVolumeClaim
+a|`pvc`
+
+|Project
+a|`project`
```

Comment:
same comment as for "namespace": Should this be really part of an application descriptor? These descriptors are stored along with other in the jar file, but the "project" or "namespace" is clearly depending on the environment where the application is installed.